Update the sign up copy and add a button to sign up

anthonydillon opened the pull request.

anthonydillon commented:
## QA
- Add an application when not signed in and on sandbox mode
- Click the deploy button
- Check the page now matches the design (without images - will add tomorrow)
- Click the sign up button and check the sign up page opens in a new tab

jujugui commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6206/
Test FAILed.

jujugui commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6207/
Test PASSed.

jujugui commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6208/
Test PASSed.

hatched commented on jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js:
```diff
@@ -79,59 +80,53 @@ YUI.add('deployment-signup', function() {
+                </ul>
+                <div className="six-col">
                   <juju.components.GenericButton
                     action={this._handleSignup}
```
This will have to be changed to an anchor tag with `target="_blank"`. The above `window.open` will be blocked by the browsers.

hatched commented on jujugui/static/gui/src/app/components/deployment-flow/signup/test-signup.js:
```diff
@@ -47,66 +47,60 @@ describe('DeploymentSignup', function() {
   });
 
-  it('can navigate to the flow view', function() {
+  xit('can navigate to the flow view', function() {
```
why is this skipped?

anthonydillon commented:
@hatched Thanks for the feedback. I have changed the button to a link and triggered the stateChange so when they return they see the demo deployment flow. Also fixed the test.

I have added the images to the page so will require a `clean-gui fast-babel run`.

jujugui commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6210/
Test PASSed.

jujugui commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6212/
Test FAILed.

jujugui commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6213/
Test PASSed.

hatched commented on jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js:
```diff
@@ -70,7 +71,14 @@ YUI.add('deployment-signup', function() {
     _handleSignup: function() {
       this._setBetaCookie(true);
-      console.error('Not implemented.');
+      this.props.changeState({
```
Do we want to immediately navigate them when they click to open the new window? I'm just wondering if it might be confusing because to them they clicked to join the beta...are they now in the beta?

anthonydillon replied:
I guess it could be confusing but the alternative is to leave them on the beta sign up page and make them find the button above or close and reopen the deployment. UX was happy with it working this way.

anthonydillon commented:
Thanks :shipit:

jujugui commented:
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge
